app/views/news/show.html.erb

```diff
@@ -27,24 +27,31 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
-<% content_for :action_menu_specific do %>
-  <%= link_to(l(:button_edit),
-        edit_news_path(@news),
-        :class => 'icon icon-edit',
-        :accesskey => accesskey(:edit),
-        :onclick => 'Element.show("edit-news"); return false;') if User.current.allowed_to?(:manage_news, @project) %>
-  <%= watcher_link(@news, User.current) %>
-  <%= link_to(l(:button_delete),
-        news_path(@news),
-        :confirm => l(:text_are_you_sure),
-        :method => :delete,
-        :class => 'icon icon-delete') if User.current.allowed_to?(:manage_news, @project) %>
+<%= toolbar title: "#{avatar(@news.author)} #{@news.title}" do %>
```

Comment:
I' d recommend removing avatars from page titles, albeit temporarily until this can be better thought out. There is a `user-avatar` component. Perhaps a `page-title` component should mirror that markup structure.